.helm-staging.yml

```diff
@@ -38,3 +36,11 @@ analyzers:
   - name: sonarcheck-analyzer
     addr: ipv4://lookout-sonarcheck-analyzer:10304
     feedback: https://github.com/src-d/lookout-sonarcheck-analyzer/issues/new
+
+rabbitmq-ha:
+  replicaCount: 1
+  rbac:
+    create: false
```

Comment:
Why is this set to false? The lookout-tiller service account should allow RBAC creation which is used by the chart to list service endpoints.

Reply:
It doesn't work with RBAC. See https://github.com/src-d/infrastructure/issues/174#issuecomment-435835131

Reply:
Thanks for the info! It's not a problem :)